### ISSUE TITLE
Maya doesn't put a card on the bottom of the deck

### ISSUE THREAD

**Issue by NorthMaester** [mtgred/netrunner#1496]
When I press Maya I get the option to put either the card currently being accessed to the bottom of the deck, or a previously accessed card.
When I press the first option whilst accessing a card, nothing happens. When I choose the other option, I get a dialouge prompting me to choose which card, but nothing I press changes anything.

I made several runs with Medium installed, and whatever I tried, the same cards were there every time.

**Comment by JoelCFC25** [comment]
So far I'm having trouble replicating this. I've tried using Maya by itself and also with a loaded Medium to bottom a card midway through a multi-access and it's working for me. Any other details you can share about cards you had installed might help--and you did remember that it's a "once per turn", right?

**Comment by queueseven** [comment]
Maya seems to work fine for me.

**Comment by NorthMaester** [comment]
I played a version of the Jumping Dogs deck, so I had a couple of Cerberus
breakers, Brahman, some NetChips, Clone Chip, Savoir-Faire, Patron, John
Masanori. As I can recall.
But I tried again yesterday with the same deck, and it seemed to work just
fine. So it seems it was just some glitch.

2016-05-10 9:38 GMT+02:00 Dominic Kexel <notifications@github.com>:

> Maya seems to work fine for me.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mtgred/netrunner/issues/1496#issuecomment-218081476>
>



-- 
Mvh
Ruben Byman
